### ISSUE TITLE
Build error with 1.4.0

### ISSUE THREAD

**Issue by raveit65** [zhuyaliang/user-admin#18]
I got an build error with building RPM from 1.4.0 tarball.
```
[14/15] cc -Isrc/25a6634@@mate-user-admin@exe -Isrc -I../src -I. -I../ -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/fribidi -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/uuid -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/gio-unix-2.0 -I/usr/include/libdrm -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/accountsservice-1.0 -I/usr/include/group-service-1.0 -I/usr/include/mate-desktop-2.0 -I/usr/include/startup-notification-1.0 -I/usr/include/dconf -I/usr/include/polkit-1 -I/builddir/build/BUILD/user-admin-1.4.0/x86_64-redhat-linux-gnu -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -std=gnu11 -DHAVE_CONFIG_H -fstack-protector-strong -g -Waggregate-return -Warray-bounds -Wcast-align -Wclobbered -Wdeclaration-after-statement -Wempty-body -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wformat-signedness -Wignored-qualifiers -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-declarations -Wmissing-format-attribute -Wmissing-include-dirs -Wmissing-noreturn -Wmissing-parameter-type -Wmissing-prototypes -Wnested-externs -Wno-discarded-qualifiers -Wno-missing-field-initializers -Wno-strict-aliasing -Wno-suggest-attribute=format -Wno-unused-parameter -Wold-style-definition -Woverride-init -Wpacked -Wpointer-arith -Wredundant-decls -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wstrict-prototypes -Wswitch-default -Wtype-limits -Wundef -Wuninitialized -Wunused-but-set-variable -Wwrite-strings -O2 -g -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -pthread  -MD -MQ 'src/25a6634@@mate-user-admin@exe/user-group.c.o' -MF 'src/25a6634@@mate-user-admin@exe/user-group.c.o.d' -o 'src/25a6634@@mate-user-admin@exe/user-group.c.o' -c ../src/user-group.c
../src/user-group.c: In function 'CreateRemoveModel':
../src/user-group.c:576:13: warning: implicit declaration of function 'gas_group_is_primary_group'; did you mean 'gas_group_is_local_group'? [-Wimplicit-function-declaration]
  576 |         if(!gas_group_is_primary_group(group->gas) &&
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~
      |             gas_group_is_local_group
../src/user-group.c:576:13: warning: nested extern declaration of 'gas_group_is_primary_group' [-Wnested-externs]
[15/15] cc  -o src/mate-user-admin 'src/25a6634@@mate-user-admin@exe/main.c.o' 'src/25a6634@@mate-user-admin@exe/user-admin.c.o' 'src/25a6634@@mate-user-admin@exe/user-base.c.o' 'src/25a6634@@mate-user-admin@exe/user-face.c.o' 'src/25a6634@@mate-user-admin@exe/user-info.c.o' 'src/25a6634@@mate-user-admin@exe/user-list.c.o' 'src/25a6634@@mate-user-admin@exe/user-password.c.o' 'src/25a6634@@mate-user-admin@exe/user-share.c.o' 'src/25a6634@@mate-user-admin@exe/user-crop.c.o' 'src/25a6634@@mate-user-admin@exe/user-group.c.o' 'src/25a6634@@mate-user-admin@exe/user-history.c.o' 'src/25a6634@@mate-user-admin@exe/user-language.c.o' -Wl,--no-undefined -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wl,-z,relro -Wl,--as-needed -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -Wl,--start-group /usr/lib64/libgtk-3.so /usr/lib64/libgdk-3.so /usr/lib64/libpangocairo-1.0.so /usr/lib64/libpango-1.0.so /usr/lib64/libatk-1.0.so /usr/lib64/libcairo-gobject.so /usr/lib64/libcairo.so /usr/lib64/libgdk_pixbuf-2.0.so /usr/lib64/libgio-2.0.so /usr/lib64/libgobject-2.0.so /usr/lib64/libglib-2.0.so /usr/lib64/libpwquality.so /usr/lib64/libaccountsservice.so /usr/lib64/libgroup-service.so -lcrypt /usr/lib64/libmate-desktop-2.so /usr/lib64/libstartup-notification-1.so /usr/lib64/libpolkit-gobject-1.so /usr/lib64/libfontconfig.so /usr/lib64/libfreetype.so -Wl,--end-group  
FAILED: src/mate-user-admin 
cc  -o src/mate-user-admin 'src/25a6634@@mate-user-admin@exe/main.c.o' 'src/25a6634@@mate-user-admin@exe/user-admin.c.o' 'src/25a6634@@mate-user-admin@exe/user-base.c.o' 'src/25a6634@@mate-user-admin@exe/user-face.c.o' 'src/25a6634@@mate-user-admin@exe/user-info.c.o' 'src/25a6634@@mate-user-admin@exe/user-list.c.o' 'src/25a6634@@mate-user-admin@exe/user-password.c.o' 'src/25a6634@@mate-user-admin@exe/user-share.c.o' 'src/25a6634@@mate-user-admin@exe/user-crop.c.o' 'src/25a6634@@mate-user-admin@exe/user-group.c.o' 'src/25a6634@@mate-user-admin@exe/user-history.c.o' 'src/25a6634@@mate-user-admin@exe/user-language.c.o' -Wl,--no-undefined -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wl,-z,relro -Wl,--as-needed -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -Wl,--start-group /usr/lib64/libgtk-3.so /usr/lib64/libgdk-3.so /usr/lib64/libpangocairo-1.0.so /usr/lib64/libpango-1.0.so /usr/lib64/libatk-1.0.so /usr/lib64/libcairo-gobject.so /usr/lib64/libcairo.so /usr/lib64/libgdk_pixbuf-2.0.so /usr/lib64/libgio-2.0.so /usr/lib64/libgobject-2.0.so /usr/lib64/libglib-2.0.so /usr/lib64/libpwquality.so /usr/lib64/libaccountsservice.so /usr/lib64/libgroup-service.so -lcrypt /usr/lib64/libmate-desktop-2.so /usr/lib64/libstartup-notification-1.so /usr/lib64/libpolkit-gobject-1.so /usr/lib64/libfontconfig.so /usr/lib64/libfreetype.so -Wl,--end-group  
/usr/bin/ld: src/25a6634@@mate-user-admin@exe/user-group.c.o: in function `CreateRemoveModel':
/builddir/build/BUILD/user-admin-1.4.0/x86_64-redhat-linux-gnu/../src/user-group.c:576: undefined reference to `gas_group_is_primary_group'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```
full log is here https://www.dropbox.com/s/6vld4ymavyfq2g4/user-admin-build-log?dl=0
I used group-service-1.0.0 tarball as dependency
I didn't notice the issue if i compiled and installed both packages by hand from git master.
Do i need an updated group-service tarball?

**Comment by zhuyaliang** [comment]
@raveit65 Please use version 1.1.0, I have resolved these errors and warnings.

**Comment by zhuyaliang** [comment]
@raveit65 
I released mate-user-admin version 1.4.1. group-service 1.1.0.You can use them to build
